layouts/shortcodes/notices/editQualityNotice.html

```diff
@@ -0,0 +1,2 @@
+{{ partial "notice.html" (merge (dict "page" .Page) (dict "noticeTitle" "Poor Edit Quality" "noticeDescription" "This page does not meet our <a>editing standards</a>. You can help by fixing it up." ) ) }}
```

Comment:
The `<a></a>` element is unnecessary imo until we actually have an editing standards article.